### PR TITLE
can follow snake tail

### PR DESCRIPTION
**Pull request by dlsteuer**

### CODE REVIEW

File: rules/tick.go [hunk 1]
```diff
@@ -30,21 +30,7 @@ func GameTick(game *pb.Game, lastFrame *pb.GameFrame) (*pb.GameFrame, error) {
...
-		"Turn":   nextFrame.Turn,
-	}).Info("check for death")
-	deathUpdates := checkForDeath(game.Width, game.Height, nextFrame)
-	for _, du := range deathUpdates {
-		if du.Snake.Death == nil {
-			du.Snake.Death = du.Death
-		}
-	}
-	// 3. game update
+	// 2. game update
```
Comment (nbw) [inline rules/tick.go]: on line 37, can we say didn't _eat_.  

I know, it's a typo from before.

File: rules/tick.go [hunk 2]
```diff
@@ -70,6 +56,21 @@ func GameTick(game *pb.Game, lastFrame *pb.GameFrame) (*pb.GameFrame, error) {
 		return nil, err
 	}
 	nextFrame.Food = nextFood
+
+	// 3. check for death
+	// 	  a - starvation
```
Comment (nbw) [inline rules/tick.go]: extraneous tab (sorry!)

**Comment by nbw** [comment]
lgtm.

File: rules/tick.go (hunk 2 above)
Comment (dlsteuer) [reply rules/tick.go]: strange not like that in my editor

**Pull request merged**